l10n/en/products/vpn/platforms/ios.ftl

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/mobile/ios
+
+# Meta strings
+vpn-ios-page-title = { -brand-name-mozilla-vpn } for { -brand-name-ios } ({ -brand-name-iphone } & { -brand-name-ipad })
+
+vpn-ios-page-description = Get fast, easy security on your { -brand-name-iphone } or { -brand-name-ipad } through { -brand-name-mozilla }, one of the most trusted names in technology.
+
+# Mobile strings
+vpn-ios-hero-headline = Protect yourself online with { -brand-name-mozilla-vpn } for { -brand-name-ios }
+
+# Variables:
+#   $devices (number) - number of available devices
+vpn-ios-ability = Feel safe from hackers and prying eyes when you’re online — both at home and out and about — with { -brand-name-mozilla-vpn }. For a low monthly fee, it uses advanced { -brand-name-wireguard }® protocol to encrypt personal data on your { -brand-name-iphone }, { -brand-name-ipad } and three more devices, letting you stream shows, play games, shop, and go about your daily life online knowing that you’re secure.
```

Comment:
Revise "three more devices" to be the maximum number of devices or some other fix.